src/components/layout/Banner/index.tsx

```diff
@@ -1,16 +1,14 @@
 import React, { FC } from 'react';
 import { useTranslation } from 'i18n';
 import { FontIcon, FontIconName } from 'components/common/FontIcon';
-import styles from './styles.module.scss';
+import styles from './main.module.scss';
 
 export const Banner: FC = () => {
-  const { t } = useTranslation();
-  return (
-    <div className={styles.banner}>
-      <FontIcon name={FontIconName.Warning} size={12} />
-      {t(
-        'Ricochet Exchange smart contracts are unaudited, use at your own risk.',
-      )}
-    </div>
-  );
+	const { t } = useTranslation();
```

Comment:
This layout banner can be removed completely, as we don't use it anymore. 

You can go ahead and delete the whole folder